docs/getting-started/first-service.md

```diff
@@ -2,9 +2,10 @@
 !!! tip
     Hit ++"n"++ / ++"."++ on your keyboard to move forward in the tutorial. Use ++"p"++ / ++","++ to go back at any time.
     
-==**In this tutorial, we are going to use [KonK](https://konk.dev) to deploy a "Hello world" Service!**== This service will accept an environment variable, `TARGET`, and print "`Hello $TARGET`."
+==**In this tutorial, we are going to deploy a "Hello world" Service!**== 
+This service will accept an environment variable, `TARGET`, and print "`Hello ${TARGET}!`."
 
-For those of you familiar with other **source-to-url** tools, this may seem familiar. However, since our "Hello world" Service is being deployed as a Knative Service, it gets some **super powers (scale-to-zero, traffic-splitting) out of the box** :rocket:.
+For those of you familiar with other **kubernetes** tools, this may seem familiar. However, since our "Hello world" Service is being deployed as a Knative Service, it gets some **super powers (scale-to-zero, traffic-splitting) out of the box** :rocket:.
```

Comment:
```suggestion
For those of you familiar with other **source-to-url** tools, this may seem familiar. However, since our "Hello world" Service is being deployed as a Knative Service, it gets some **super powers (scale-to-zero, traffic-splitting) out of the box** :rocket:.
```

I really was going for "source-to-url" here, trying to keep as much kubernetes language out of the tutorial as possible. It's also a very common pattern to do source -> url